validator/client/validator_attest.go

```diff
@@ -79,7 +79,6 @@ func (v *validator) AttestToBlockHead(ctx context.Context, slot uint64, pubKey [
 
 	aggregationBitfield := bitfield.NewBitlist(uint64(len(assignment.Committee)))
 	aggregationBitfield.SetBitAt(indexInCommittee, true)
-	log = log.WithField("bitfield", fmt.Sprintf("%08b", aggregationBitfield))
```

Comment:
Why?

Reply:
This is not in master, i briefly added for v0.9 branch for debugging and sanity checking the logs to gain confidence every validator is correctly attesting. If you want we can keep it. It does make the log kinda messy : )